data/p10/FAPITargetsNameMapList.lsv

```diff
@@ -21,9 +21,9 @@ TARGET_TYPE_MI:unit-mi-power10
 TARGET_TYPE_MCC:unit-mcc-power10
```

Comment:
Commit message header: can you please add a prefix like below

`data: p10:  Changes to support new DIMM and OCMB MRW target types`